lib/ramen_bar.rb

```diff
@@ -0,0 +1,34 @@
+class RamenBar
+
+  attr_accessor :dishes, :order, :total_order
+
+  MENU = {
```

Comment:
Good simple usage of the hash here, definitely overcomplicated mine with an array of hashes!